doc/classes/TileMap.xml

```diff
@@ -156,20 +156,20 @@
 			</argument>
 			<argument index="5" name="transpose" type="bool" default="false">
 			</argument>
-			<argument index="6" name="autotile_coord" type="Vector2" default="Vector2( 0, 0 )">
+			<argument index="6" name="tile_coordinate" type="Vector2" default="Vector2( 0, 0 )">
 			</argument>
 			<description>
 				Sets the tile index for the cell given by a Vector2.
 				An index of [code]-1[/code] clears the cell.
-				Optionally, the tile can also be flipped, transposed, or given autotile coordinates. The autotile coordinate refers to the column and row of the subtile.
+				Optionally, the tile can also be flipped, transposed, or given tile coordinate. The tile coordinate refers to the column and row of the subtile.
 				[b]Note:[/b] Data such as navigation polygons and collision shapes are not immediately updated for performance reasons.
 				If you need these to be immediately updated, you can call [method update_dirty_quadrants].
 				Overriding this method also overrides it internally, allowing custom logic to be implemented when tiles are placed/removed:
 				[codeblock]
-				func set_cell(x, y, tile, flip_x, flip_y, transpose, autotile_coord)
+				func set_cell(x, y, tile = false, flip_x = false, flip_y = false, transpose = false, tile_coordinate = Vector2())
```

Comment:
I changed this because now we have typed gdscript, without passing the default values ​​it will not work.